src/effects/index.ts

```diff
@@ -85,5 +86,15 @@ export const CHAT_EFFECTS: Array<Effect<{ [key: string]: any }>> = [
             gravity: 0.01,
         },
     } as Effect<SpaceInvadersOptions>,
+    {
+        emojis: ["💝", "💖", "💞", "💕"],
+        msgType: "io.element.effect.hearts",
+        command: "hearts",
+        description: () => _td("Sends the given message with hearts"),
+        fallbackMessage: () => _t("sends hearts") + " 💝",
```

Comment:
This looks great!

One nit to merge— we don't really see much use of 💝 in the wild. Please can you amend to use 💞 in the timeline emote/message? Thanks!

Reply:
Personally I think gift_heart is more fitting in this context, but I've amended it anyway.